S2VX.Game/Editor/Containers/NotesTimeline.cs

```diff
@@ -261,7 +261,7 @@ public class NotesTimeline : CompositeDrawable {
             var upperBound = Time.Current + sectionDuration / 2;
             foreach (var note in Story.Notes.Children) {
                 RelativeBox visibleNote = null;
-                var color = note.GetColor();
+                var color = note.InnerColor();
```

Comment:
This was renamed because of the C# guidelines to not start things with "Get" unless it's a getter for a property right

Reply:
Yeah, for more context: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1024